lava_test_plans/testcases/master/template-mmtests.jinja2

```diff
@@ -0,0 +1,20 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% set KERNEL_BRANCH = KERNEL_BRANCH|default("") %}
+{% set OS_INFO = OS_INFO|default("") %}
+
+{% set test_name = "mmtests-" + config_file|replace('configs/config-', '') %}
+
+{% block test_target %}
+  {{ super() }}
+{% block test_target_prep %}
+{% endblock test_target_prep %}
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/mmtests/mmtests.yaml
+      parameters:
+        SKIP_INSTALL: 'true'
+        MMTESTS_CONFIG_FILE: '{{ config_file }}'
+        MMTESTS_ITERATIONS: '{{ mmtests_iterations }}'
+      name: mmtests-{{ config_file|replace('configs/config-', '')  }}
+{% endblock test_target %}
```

Comment:
Please change this file like the other template-testsuite files in this directory template-kselftest.yaml.jinja2 template-ltp.yaml.jinja2.